### ISSUE TITLE
MD037 triggering for escaped astricks in math equations

### ISSUE THREAD

**Issue by nschonni** [DavidAnson/markdownlint#168]
I'm seeing this triggered from through the Markdownlint (0.12) invoke through Markdownlint-CLI (0.14) 
```
Log((Sign(x) \* Sqrt(x \* x + 1) + 1) / x)
```
When I open the file in VS Code with the Markdownlint extension, it isn't picked up as an error though

**Comment by DavidAnson** [comment]
This is an issue for igorshubovych/markdownlint-cli which doesn't (yet) recognize math blocks. See https://github.com/DavidAnson/vscode-markdownlint/commit/bd71955cf99bc0ace103b7b7fa121435446f5e32 for how this was implemented in the VS Code extension. I'd thought to confirm it worked correctly in the VS Code extension before updating the CLI.

(I'd transfer the issue, but don't seem to be able to do so into someone else's namespace.)

**Comment by nschonni** [comment]
The code itself isn't actually in a code fence so I thought it might be a regex issue here

**Comment by DavidAnson** [comment]
I assumed you just didn’t include the fence. I can have a closer look soon.

**Comment by nschonni** [comment]
Here is one of the files I've hit it on https://github.com/dotnet/docs/blob/master/docs/visual-basic/language-reference/keywords/derived-math-functions.md
Note that it does have some unescaped `*`, but I still get the warnings after escaping all the asterisks in the file

**Comment by DavidAnson** [comment]
I think this is unrelated to the new math block handling and is a bug in this library. Is there a chance the rule is disabled in your VS Code configuration? I’d expect to see the error reported there as well (though I can’t verify that right now).

**Comment by nschonni** [comment]
I don't have any rules set in my VS Code, but I do have the Docs Authoring Extension that adds these rules https://github.com/Microsoft/vscode-docs-authoring/tree/master/docs-markdown/markdownlint-custom-rules

**Comment by DavidAnson** [comment]
Okay, so this is not showing up in VS Code because the range that markdownlint emits for the error causes an exception when reported by the extension because the starting character is out of range. That breaks linting and nothing else is output. Once fixed in markdownlint, this should start working in the VS Code extension.
